src/sass/partials/layouts/_content-card.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 2rem 0 0;
+  width: 100%;
+
+
+  @include breakpoint(setting-get('two grid cards')) {
+    margin-right: 4%;
+    width: 48%;
+
+    &:nth-child(2n) {
+      margin-right: 0;
+    }
+  }
+
+  @include breakpoint(setting-get('fixed container')) {
+    margin-right: 4%;
+    width: 22%;
```

Comment:
Again, as above, I'm assuming this is… 25%? With a 3% margin on each side? I'm not sure that math actually works out given the numbers here, but either way, previous comment